federator-env/mainnet-ETH-RSK/config.js

```diff
@@ -217,7 +217,43 @@ module.exports = {
                     "amount": 100000000000000000000,
                     "confirmations": 50
                 }
-            ],	
+            ],
+            "BOS": [
+                {
+                    "amount": 0,
+                    "confirmations": 10
+                },
+                {
+                    "amount": 1000000000000000000,
+                    "confirmations": 20
+                },
+                {
+                    "amount": 10000000000000000000,
+                    "confirmations": 30
+                },
+                {
+                    "amount": 100000000000000000000,
+                    "confirmations": 50
+                }
+            ],
+            "BOSes": [
```

Comment:
if the token symbol will be replace to "BOS" by the new TokenFactory, do you think it still makes sense to keep the BOSes?